back/lib/participation_method/native_survey.rb

```diff
@@ -13,7 +13,7 @@ def allowed_extra_field_input_types
 
     def assign_defaults(input)
       input.publication_status ||= 'published'
-      input.idea_status ||= IdeaStatus.find_by!(code: 'proposed')
+      input.idea_status ||= IdeaStatus.find_by!(code: 'proposed', participation_method: 'ideation')
```

Comment:
It's possible that some native survey responses that were recently submitted were assigned to the proposed status for proposals instead. Do you think we should spend time writing a script to reassign them to the proposed status for ideation?

Reply:
Seems like [there aren't any](https://metabase.hq.citizenlab.co/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7ImRhdGFiYXNlIjozLCJ0eXBlIjoicXVlcnkiLCJxdWVyeSI6eyJzb3VyY2UtdGFibGUiOjIwNjYsImpvaW5zIjpbeyJmaWVsZHMiOiJhbGwiLCJhbGlhcyI6IklkZWEgU3RhdHVzZXMiLCJjb25kaXRpb24iOlsiPSIsWyJmaWVsZCIsODkxOTYseyJiYXNlLXR5cGUiOiJ0eXBlL1RleHQifV0sWyJmaWVsZCIsOTI3MDcseyJiYXNlLXR5cGUiOiJ0eXBlL1RleHQiLCJqb2luLWFsaWFzIjoiSWRlYSBTdGF0dXNlcyJ9XV0sInNvdXJjZS10YWJsZSI6MjAwMn1dLCJmaWx0ZXIiOlsiPSIsWyJmaWVsZCIsMTYxMzA0LHsiYmFzZS10eXBlIjoidHlwZS9UZXh0Iiwiam9pbi1hbGlhcyI6IklkZWEgU3RhdHVzZXMifV0sInByb3Bvc2FscyJdfX0sImRpc3BsYXkiOiJ0YWJsZSIsInZpc3VhbGl6YXRpb25fc2V0dGluZ3MiOnt9fQ==), so we should be good